thaomonster/ACNH-Collections
# Feature/api

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds the api call.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

` src/apiCalls.js `lines: 1-15;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2.  open the code and look at the apiCalls file.


### Steps To Take
>Any questions and possible steps moving forward.

Make it appear on the webpage.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #6 & #7
